terraform/aws/modules/customer-web-server/cws_compliance.tf

```diff
@@ -0,0 +1,50 @@
+resource "kubernetes_cron_job" "cws_compliance_cron" {
+  count = var.deploy_cws_compliance_cronjob ? 1 : 0
+
+  metadata {
+    name      = "cws-compliance-lookup"
+    namespace = var.cws_name
+  }
+  spec {
+    concurrency_policy            = "Forbid"
+    failed_jobs_history_limit     = 2
+    successful_jobs_history_limit = 2
```

Comment:
@stylianosrigas vvvvvvvv minor But do we actually need more than 1?

Reply:
Thanks makes sense. It runs only every 12 hours so we don't need more that one ;)